framework/src/org/checkerframework/common/value/util/Range.java

```diff
@@ -20,6 +20,9 @@
     /** The upper bound of the interval, inclusive. */
     public final long to;
 
+    /** Should ranges take overflow into account or ignore it? */
```

Comment:
We avoided doing this because it results in the `Range` class becoming quite bloated: most Range operations produce another Range, so the value of `ignoreOverflow` as an instance field would need to be passed through.

With the current design, any checker that uses the `Range` library should set the value of the `IGNORE_RANGE_OVERFLOW`. This permits reuse. That said, I agree that it's undesirable to force clients to set that field because it allows for the possibility of error.

I view both designs as having non-trivial disadvantages. Do you believe we should switch back to an instance-field design? @smillst, did I forget an important argument for why this should be a static field?